lib/task_listener.js

```diff
@@ -39,6 +30,12 @@ export default class TaskListener extends EventEmitter {
     this.runningTasks = [];
     this.taskQueue = new TaskQueue(this.runtime);
     this.taskPollInterval = this.runtime.taskQueue.pollInterval;
+
+    if (this.runtime.isolatedContainers) {
```

Comment:
Hmm... I see how `isolatedContainers` might make more sense longer down...

Maybe, just set `deviceManagement.cpu.enabled = false` in `config/defaults.js`, then the code here only needs to be `deviceManagement.cpu.enabled = true`, which is a bit less obscure than the object assignment...

Maybe, not sure... probably just a detail.